word-count/pipeline.yaml

```diff
@@ -1,19 +1,21 @@
-- type: producer-app-v2
+- type: producer-app
   name: data-producer
   values:
     image: bakdata/kpops-demo-sentence-producer
+    imageTag: "2.0.0"
 
-- type: streams-app-v2
-  name: word-counter
+- type: streams-app
+  name: word-count-app
+  values:
+    image: bakdata/kpops-demo-word-count-app
```

Comment:
this depends on #19 to have it in a public Docker registry, right?

Reply:
That is true. I haven't changed anything in the `pipeline.yaml` other than the migration to streams-bootstrap v3. We should upload the new images manually.

Reply:
I wonder how much effort it would be to just finish the CI PR. it would be really convenient to automatically upload those images.